### DISCUSSION TITLE
테스트에러 TypeError: Cannot read property 'add_RoomCreated' of undefined

### DISCUSSION THREAD

**Discussion by heavybowgun** [naverz/zepeto-studio-kor#1180] [5. Q&A]
현재 멀티 샘플예제에서 오브젝트만 지우고 그 위에서 작업을 진행하고 있습니다.

작업하며 계속 테스트플레이눌러서 잘 구동하는걸 확인했는데 갑자기 

TypeError: Cannot read property 'add_RoomCreated' of undefined 이런 에러가 뜨면서 플레이가 안됩니다.

스크립트관련해선 건들지 않고 오브젝트만 배치했는데 왜 이런 문제가 생기는지 몰라서 질문 드립니다.
![0816오후2](https://user-images.githubusercontent.com/107227678/184852056-ae5a1087-abbf-4b94-9069-48c1dde676aa.JPG)
![0816오후8](https://user-images.githubusercontent.com/107227678/184852363-ce593922-3d82-43d2-bb11-f0b8a59222a7.JPG)

**Comment by hwisulee** [comment]
저도 다른 업무 하다가 다시 제페토 잡았을 때 저런 오류가 떠서 의문이었습니다.
문제점이었던 스크립트가 존재하는 오브젝트를 제거해도 문제가 생기길래 구석구석 찾아보다 기억해냈습니다.
다른 업무를 시작하기 전 구석에 박아둔 한 오브젝트에 넣어둔 동일한 스크립트의 public multiplay에 할당된 내용이 없기에 발생한 문제였습니다.

해당 스크립트가 들어간 곳을 찾으셔서 인스펙터 창에서 Zepeto World Multiplay를 끌어서 넣어주시면 사라질 것 같습니다.
